hackathon_site/dashboard/frontend/src/components/general/Footer/Footer.js

```diff
@@ -10,7 +10,7 @@ const Footer = () => {
     const outputMessage =
         "© 2020" +
         yearRange +
-        ". Built by the web team at MakeUofT (University of Toronto)";
+        " Built by the web team at MakeUofT (University of Toronto)";
```

Comment:
Can we change this to "IEEE University of Toronto Student Branch"? Or otherwise use `hackathonName` from `constants.js`, but I think it should be the club name

Reply:
The main intent was to plug our hackathon when other people fork it

Reply:
Hmm I see. I think in that case people are just going to remove it, whereas if it is our club they might keep it